web/src/alert_words.ts

```diff
@@ -41,6 +41,8 @@ export function process_message(message: Message): void {
         return;
     }
 
+    my_alert_words.sort((a, b) => b.length - a.length);
+
```

Comment:
Sorting seems the right solution but moving this line up into `set_words` will avoid having to sort words for every message.

Also, a prep commit where you call `set_words` from `initialize` would help avoid code duplication

Reply:
Thank you @N-Shar-ma , Just to clarify, calling `set_words` from `initialize`, should be done in a separate commit?

Reply:
@Danieldotcomcoder Correct, that change should be in a refactor commit, before the commit that adds the sorting.

Reply:
@N-Shar-ma Don't you think if we sort `alert_words` in `set_words` function that will change the order of `alert_words` and thus change the order of `get_words_list` ?

Is order of `alert_words` not some to bother ? If yes, then I have a PR raised for this: #29088